cobalt/network/url_request_context.cc

```diff
@@ -203,6 +204,11 @@ URLRequestContext::URLRequestContext(
               new ProxyConfigService(proxy_config)),
           net::NetLog::Get(), /*quick_check_enabled=*/true));
 
+  auto quic_context = std::make_unique<net::QuicContext>();
+  quic_context->params()->supported_versions =
+      quic::ParsedQuicVersionVector{quic::ParsedQuicVersion::Q046()};
+  url_request_context_builder->set_quic_context(std::move(quic_context));
```

Comment:
Doesn't DefaultSupportedQuicVersions() in quic_context.h override that to allow RFCv1 only? That function is where I made the change to select it.

Reply:
DefaultSupportedQuicVersions() are called in line 207 when the Context is constructed, and we overwrite the vector in 208. I verified that the client svc-vec only contains "Q046".